src/main/java/Main.java

```diff
@@ -40,9 +44,21 @@ public class Main {
     private static final String GENERATOR_STANDARD_FILES_PATH = "src/main/resources/standard_files";
 
     public static void main(String[] args) throws IOException {
-
+        File file = new File("./generated-application");
```

Comment:
In den Zeilen 68ff. ist bereits ein Bereich mit Kommentar, Basic logging und einem ToDo für das einlesen der Struktur. Bitte verschieb den Teil entsprechend dahin. Außerdem wird bereits an zentraler Stelle nachgeguckt, ob das Projekt zum ersten Mal generiert wird oder es eine wiederholte generierung ist. Das kannst du einfach nutzen